### PR TITLE
pkp/pkp-lib#5917 Simplify logo display

### PR DESCRIPTION
**Pull request by Vitaliy-1**

### CODE REVIEW

**Comment by Vitaliy-1** [comment]
On the second thought, looking at the application-specific part of code, it's better to move it all to the shared library. I'll update this PR accordingly.

**Comment by Vitaliy-1** [comment]
(don't merge this one yet)

**Comment by Vitaliy-1** [comment]
Done!

File: templates/frontend/components/header.tpl [hunk 1]
```diff
@@ -50,16 +50,12 @@
...
 							<img src="{$publicFilesDir}/{$displayPageHeaderLogo.uploadName|escape:"url"}" width="{$displayPageHeaderLogo.width|escape}" height="{$displayPageHeaderLogo.height|escape}" {if $displayPageHeaderLogo.altText != ''}alt="{$displayPageHeaderLogo.altText|escape}"{/if} />
 						</a>
-					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_string($displayPageHeaderTitle)}
+					{elseif $displayPageHeaderTitle}
 						<a href="{$homeUrl}" class="is_text">{$displayPageHeaderTitle|escape}</a>
-					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_array($displayPageHeaderTitle)}
-						<a href="{$homeUrl}" class="is_img">
-							<img src="{$publicFilesDir}/{$displayPageHeaderTitle.uploadName|escape:"url"}" alt="{$displayPageHeaderTitle.altText|escape}" width="{$displayPageHeaderTitle.width|escape}" height="{$displayPageHeaderTitle.height|escape}" />
-						</a>
 					{else}
```
Comment (NateWr) [inline templates/frontend/components/header.tpl:59]: I think this last `{else}` statement is not needed, right? There will always be a `$displayPageHeaderTitle`. If so, you can get rid of this last black and make the previous `{elseif}` an `{else}`.

Comment (Vitaliy-1) [reply templates/frontend/components/header.tpl:59]: I didn't change the logic here. If the logo and title are absent, the OJS logo is shown instead. There may be a situation when the site name is empty.

Comment (NateWr) [reply templates/frontend/components/header.tpl:59]: Ah ok, I didn't realize the site name could be empty. :+1:

Comment (Vitaliy-1) [reply templates/frontend/components/header.tpl:59]: @NateWr, just checked and this applies to the current master branch too: when the second journal is created, a user isn't prompted to a site-level settings page but the page becomes visible on the front-end. Maybe it's a place for improvement?

Comment (NateWr) [reply templates/frontend/components/header.tpl:59]: Yeah, my sense is that a site title should be entered on installation. But not a priority for now.

**Comment by Vitaliy-1** [comment]
@NateWr, I did a minor modification of the existing PR, all tests are passing and I think it's ready to be merged!

**Pull request merged**